VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/hearing-rules/hearing-rules.component.html

```diff
@@ -7,9 +7,13 @@ <h1 class="govuk-heading-xl">{{ 'hearing-rules.court-rules' | translate }}</h1>
   <div class="govuk-grid-column-one-quarter">
     <div class="image">
       <img
-        [src]="(hearingVenueIsScottish$ | async) ? '/assets/images/scottish-crest-circle.png': '/assets/images/HmctsCrest.png'"
+        appCrestLogoImageSource
+        [sctsImageSource]="'/assets/images/scottish-crest-circle.png'"
+        [sctsAltText]=" ('hearing-rules.scottish-crest' | translate)"
+        [hmctsImageSource]="'/assets/images/HmctsCrest.png'"
+        [hmctsAltText]=" ('hearing-rules.hm-courts-and-tribunals-service-crest' | translate)"
         class="govuk-header__logotype-crest crest"
-        [attr.alt]="(hearingVenueIsScottish$ | async) ? ('hearing-rules.scottish-crest' | translate): ('hearing-rules.hm-courts-and-tribunals-service-crest' | translate)"
+        alt=""
```

Comment:
Why does this need to be here?

Reply:
sonarcloud flags this as a bug even though we are adding the alt text using javascript